src/localstack.js

```diff
@@ -24,14 +24,19 @@ class TempWriteBuffer extends Writable {
   }
 
   _write (chunk, encoding, callback) {
```

Comment:
I'm guessing these changes are also revertable after pulling master

Reply:
I don't know what you mean?

Reply:
In this case and the code in `src/lambda.js` it seemed like you were changing things other than "Add an iterator to be a kinesis trigger"

At first glance,  it looked like the changes were related to the build issues I fixed in master. My hunch was that you could simplify a lot of the PR and reduce it down to just changes needed for Kinesis. Perhaps there are other issues that you found which are also getting mixed in here.  If these changes are really needed for Kinesis, I can try re-review